tests/test_generators/test_docgen.py

```diff
@@ -322,64 +325,243 @@ def test_docgen(kitchen_sink_path, input_path, tmp_path):
     assert len(set(domain_of_species_name)) == len(domain_of_species_name)
 
 
-def test_docgen_multiline_description_hierarchical_view(tmp_path) -> None:
```

Comment:
I replaced this test, which checks a couple of multi-line description scenarios, with comprehensive tests for most? all? the different elements that can have multi-line descriptions. Note that multi-line descriptions are only a problem in the markdown tables (where the `enshorten` filter is used), so that's specifically what I'm testing.